railties/lib/rails/generators/rails/app/templates/Gemfile

```diff
@@ -36,12 +36,21 @@ group :development, :test do
 end
 
 group :development do
+<%- if options.api? -%>
+  # Access an IRB console on exception pages
+  <%- if options.dev? || options.edge? -%>
+  # gem 'web-console', github: 'rails/web-console'
```

Comment:
Since we decided to have a separate config value (`debug_exception_response_format`) to setup the debug response format, it's still valid to use `web-console` in an API only app. You can configure `debug_exception_response_format = :default` and `api_only = true`, so you should get HTML error pages on development where the console can be used.

In the other hand, JSON responses are the default configuration (`debug_exception_response_format = :api`), and for this reason it makes sense to do not include `web-console` by default. However, we're leaving it commented because it's still supported.

In a related note, we were waiting for changes in `web-console` to fully support this case (https://github.com/rails/web-console/pull/152), but that's already merged :)